contracts/JBTiered721DelegateStore.sol

```diff
@@ -401,22 +354,20 @@ contract JBTiered721DelegateStore is IJBTiered721DelegateStore {
     return
       JB721Tier({
         id: _tierId,
-        contributionFloor: _storedTier.contributionFloor,
-        lockedUntil: _storedTier.lockedUntil == 0
-          ? 0
-          : _BASE_LOCK_TIMESTAMP + _storedTier.lockedUntil,
+        price: _storedTier.price,
         remainingQuantity: _storedTier.remainingQuantity,
         initialQuantity: _storedTier.initialQuantity,
-        votingUnits: _storedTier.votingUnits,
+        votingUnits: _storedTier.useVotingUnits ? _storedTier.votingUnits : _storedTier.price,
         // No reserved rate if beneficiary is not set.
         reservedRate: _reservedTokenBeneficiary == address(0) ? 0 : _storedTier.reservedRate,
         reservedTokenBeneficiary: _reservedTokenBeneficiary,
-        royaltyRate: _storedTier.royaltyRate,
-        royaltyBeneficiary: _resolvedRoyaltyBeneficiaryOf(_nft, _tierId),
         encodedIPFSUri: encodedIPFSUriOf[_nft][_tierId],
         category: _storedTier.category,
         allowManualMint: _storedTier.allowManualMint,
-        transfersPausable: _storedTier.transfersPausable
+        transfersPausable: _storedTier.transfersPausable,
+        resolvedUri: tokenUriResolverOf[_nft] == IJBTokenUriResolver(address(0))
+          ? ''
+          : tokenUriResolverOf[_nft].getUri(_generateTokenId(_tierId, 0))
```

Comment:
`tierOfTokenId(...)` gets called twice on every transfer (once before and once after the transfer), calling a resolver for the URI could make it expensive to the point of being unusable. We might want to add a new function that returns only the things the transfer needs instead of calling this function.

Reply:
nice

Reply:
<img width="431" alt="image" src="https://github.com/jbx-protocol/juice-721-delegate/assets/77952627/1806b16d-4ec7-46fb-9212-1735e78a4c86">
thoughts @xBA5ED ?